### PR TITLE
Add blogpost and report for fuzzing audit

### PR DESCRIPTION
**Pull request by AdamKorcz**
Adds the blogpost and report for Knative's fuzzing audit done in collaboration with CNCF and Ada Logics.

The blogpost and report are dated to be published on Monday 17th July 2023.

cc @evankanderson

### CODE REVIEW

File: blog/docs/events/fuzzing-audit-2023.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Google Summer of Code 2023 at Knative
```
Comment (psschwei) [inline blog/docs/events/fuzzing-audit-2023.md:1]: Title should probably be something about fuzzing :smile:

**Review by psschwei — COMMENTED** [review]
small nit about the title, but otherwise looks good from a docs perspective

**Comment by evankanderson** [comment]
/lgtm

I'm not sure I can approve this anymore...

@csantanapr or @lance ?

File: blog/docs/events/fuzzing-audit-2023.md (hunk 1 above)
Comment (AdamKorcz) [reply blog/docs/events/fuzzing-audit-2023.md]: Thanks 😄

**Review by psschwei — APPROVED** [review]
/approve

**Pull request merged**